castai/resource_rebalancing_schedule.go

```diff
@@ -220,11 +242,22 @@ func stateToSchedule(d *schema.ResourceData) (*sdk.ScheduledrebalancingV1Rebalan
 		if err != nil {
 			return nil, fmt.Errorf("parsing selector: %w", err)
 		}
+
+		var executionConditions *sdk.ScheduledrebalancingV1ExecutionConditions
```

Comment:
skip some boilerplate here with `toSection`, which does the cast & selection of the first element

Reply:
Wanted to use that one, but I don't think it supports returning nested blocks. So doing something like:

```
toSection(d, "launch_configuration.execution_conditions")
```

won't return the `execution_conditions`